HIPs/hlf-operator.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Hyperledger Fabric Operator Proposal
+parent: Hyperledger Improvement Proposals
+---
+
+# HIP Identifier
+[HIPs/hlf-operator.md](./hlf-operator.md)
+
+# Sponsor(s)
+
+- David Viejo [dviejo@kfs.es](mailto:dviejo@kfs.es)
+- Baohua Yang [baohua.yang@oracle.com](mailto:baohua.yang@oracle.com)
+- Aditya Joshi [adityaprakashjoshi1@gmail.com](mailto:adityaprakashjoshi1@gmail.com)
+
+# Abstract
+
+HLF Operator is a Kubernetes operator that helps simplify the deployments of Hyperledger Fabric in Kubernetes. It is based on the Hyperledger Fabric Helm charts and provides a Kubernetes native way to deploy and manage Hyperledger Fabric networks.
+
+# Context
+
+There are many ways to deploy Hyperledger Fabric, but most of the alternatives focus on Docker and manually performed deployments, not to mention there's no support to perform operations like:
+
+- Renewal of the certificates
+- Upgrades
+- Channel management
+
+HLF-operator solves this by introducing the concept of the Kubernetes operator, which is a program that runs within Kubernetes and extends it with functionality specific to HLF that makes managing networks way easier than other alternatives such as Docker-based.
+
+# Dependent Projects
+
+Hyperledger Fabric Operator only requires **Hyperledger Fabric** to run.
+
+# Motivation
+
+When building Hyperledger Fabric networks, there are several barriers to starting using and deploying Hyperledger Fabric:
+
+- How do I deploy a network in Kubernetes?
+- How do I manage the certificate authorities/peers/orderers in a production environment?
+- How do I renew the certificates?
+
+Several projects try to achieve the same goal, such as:
```

Comment:
Is there a study on how other existing projects compare to the proposal? For instance, you could check the way Bevel team approached in their proposal to the TOC https://github.com/hyperledger/hyperledger-hip/blob/gh-pages/HIPs/bevel.md#how-does-hyperledger-bevel-relate-to-existing-projects-and-labs.

Reply:
Not ATM, in the documentation of hlf-operator, in the introduction, there's a draft that I wrote at the start of the project, but nothing elaborated:
![image](https://user-images.githubusercontent.com/84456338/209994510-5b485ba4-a999-42e5-a409-d568b2d37604.png)